recipes/b2/portable/conandata.yml

```diff
@@ -1,10 +1,13 @@
 sources:
   "4.3.0":
-    sha256: 97eb98343da636c314edccb08173e25cea13b155233ebb4d74e065d55f791364
-    url: https://github.com/boostorg/build/archive/4.3.0.tar.gz
+    sha256: 138c90b66edb0a28e225705a2cbf897a8cef5f87e68befc748f8e6808e21628a
+    url: https://github.com/bfgroup/b2/archive/4.3.0.tar.gz
   "4.4.0":
-    sha256: b414592b25679ec356b96d50940d5f458fb1c6799dd10a39ac6f2e3974cc5431
-    url: https://github.com/boostorg/build/archive/4.4.0.tar.gz
+    sha256: fa4079370644110604895ff08057fe2eff3289bcffdaec55fcdf43ea33ff25a8
+    url: https://github.com/bfgroup/b2/archive/4.4.0.tar.gz
   "4.4.1":
-    sha256: 31a243b1eb26638500977a8386e56d44f86c18db70cf0a5dcdd2d7391afc1a61
-    url: https://github.com/boostorg/build/archive/4.4.1.tar.gz
+    sha256: 4fb15abd994968a24868c13502f080c4a28b20f59831acf9eabd64a3b257554f
+    url: https://github.com/bfgroup/b2/archive/4.4.1.tar.gz
+  "4.4.2":
+    sha256: 575e59b89191a6a6780d7165cae3222b8a7a1e7d5e8165b3524eefe32fc3de46
+    url: https://github.com/bfgroup/b2/archive/4.4.2.tar.gz
```

Comment:
Why did sha256 change?
And could you please provide some official confirmation, that the project is moving.

Reply:
Thanks, @grafikrobot 👍